CHANGELOG.md

```diff
@@ -14,7 +58,7 @@ FIXES:
 
 * [31](https://github.com/nginxinc/nginx-ingress-operator/pull/31) Add Status update for VS/VSR to RBAC.
 
-KNOWS ISSUES:
+KNOWN ISSUES:
```

Comment:
shall we copy this known issues section to the 0.1.0 entry? Since we didn't resolve it yet